python/common/org/python/types/Float.java

```diff
@@ -737,6 +777,84 @@ public void __delitem__(org.python.Object other) {
         throw new org.python.exceptions.TypeError("'float' object does not support item deletion");
     }
 
+    @org.python.Method(
+        __doc__ = ""
+    )
+    public org.python.types.Tuple as_integer_ratio() {
```

Comment:
This method isn't referenced anywhere else in this PR - so what is it for?

It's also not a method that is on Python integers (as far as I can tell), so assuming it *is* required, there's no need for it to be annotated for exposure into the Python namespace.

Reply:
`float.as_integer_ratio()` is a part of the **Python standard library for Floats, but not for Integers**. So if this function is not exposed to the Python namespace, the following code would execute on Python and fail on Voc.
```python
print((3.5).as_integer_ratio()) # prints (7, 2)
print((3).as_integer_ratio()) # throws an error
```
So I do believe that this function should be a part of the PR, I have checked that the diff is empty in my implementation between CPython and Voc runs. Tell me if I am wrong / should add tests for this.

Reply:
Huh - TIL. I didn't know about this one, and for some reason, it didn't come up when I did an autocomplete check in iPython (i'm guessing I mistyped the name of the method). 

I did some digging, and it appears [`as_integer_ratio()` *will* be on integers in Python 3.8](https://docs.python.org/3.8/library/stdtypes.html?highlight=as_integer_ratio#int.as_integer_ratio) - but we don't need to handle that yet.

So yes - it should definitely exist, and yes, there should be tests.